c/enc/encode.c

```diff
@@ -437,10 +437,11 @@ static BROTLI_BOOL ShouldCompress(
     if ((double)num_literals > 0.99 * (double)bytes) {
       uint32_t literal_histo[256] = { 0 };
       static const uint32_t kSampleRate = 13;
+      static const double invKSampleRate = 1.0 / 13.0;
       static const double kMinEntropy = 7.92;
       const double bit_cost_threshold =
-          (double)bytes * kMinEntropy / kSampleRate;
-      size_t t = (bytes + kSampleRate - 1) / kSampleRate;
+          (double)bytes * kMinEntropy * invKSampleRate;
+      size_t t = (bytes + kSampleRate - 1) * invKSampleRate;
```

Comment:
```suggestion
      size_t t = (double)(bytes + kSampleRate - 1) * invKSampleRate;
```

Reply:
Sorry to bother again, our internal linter is pickier on implicit conversions...